doc/source/serve/multi-app.md

```diff
@@ -83,21 +85,29 @@ $ serve deploy config.yaml
 > Sent deploy request successfully!
 ```
 
-Query the applications at their respective endpoints, `/calculator` and `/greet`.
-```pycon
->>> requests.post("http://localhost:8000/calculator", json=["ADD", 5]).json()
-7
+Query the applications at their respective endpoints, `/classify` and `/translate`.
+```python
```

Comment:
Should use [testcode](https://docs.ray.io/en/master/ray-contribute/writing-code-snippets.html#code-output-style-examples) here. But maybe better to just move those to the .py file. 

Same goes to the rest of this file.

Reply:
Changed to using test code!